### PR TITLE
Clean up and refactor into web-lib module

### PR DESCRIPTION
**Pull request by purplecabbage**
## Description

This basically removes all the old calls, in favor of simpler version and focused only on building and deploying of web-assets.
This should become a new module ( aio-lib-web ? ) and do what aio-lib-runtime does for actions/runtime only for static assets, and cdn.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by moritzraho** [comment]
first review lgtm, still missing tests

**Comment by purplecabbage** [comment]
@Himavanth Please have a look at this.

File: index.js [hunk 1]
```diff
@@ -24,58 +22,13 @@ const UndeployUI = require('./scripts/undeploy.ui')
 /**
  * @typedef AppScripts
  * @type {object}
- * @property {function(object):Promise<undefined>} buildUI - bundles the application's static files
- * @property {function(object):Promise<string>} deployUI - deploys the static files to a CDN, returns the URL
- * @property {function(object):Promise<undefined>} undeployUI - removes the deployed static files
+ * @property static {function(object):Promise<undefined>} buildWeb - bundles the application's static files
```
Comment (Himavanth) [inline index.js:25]: `static` typo ?

**Review by Himavanth — APPROVED** [review]
LGTM. Just a single probable typo.

**Pull request merged**